grails-app/services/com/recomdata/security/AuthUserDetailsService.groovy

```diff
@@ -26,6 +26,8 @@ class AuthUserDetailsService implements GrailsUserDetailsService {
        autoinjection into services */
     @Resource
     def grailsApplication
+    @Resource
+    def loginAttemptCacheService
```

Comment:
`@Resource` is not necessary; Grails already does traditional (non-BeanPostProcessor based) autowire-by-name in the domain/controller/service classes put under grails-app.